### DISCUSSION TITLE
What are the reasons for using a text-based format instead of a structured format like JSON for data field in notifications table ?

### DISCUSSION THREAD

**Discussion by a21ns1g4ts** [laravel/framework#46981] [General]
The 'data' field in the notifications schema has been defined as text. While text format is suitable for storing simple strings, developers may try to send JSON queries in the 'data' field, which can cause unexpected errors in Postgres.

**Comment by ankurk91** [comment]
Feel free to modify column in your projects, it should work fine

**Reply by a21ns1g4ts** [reply]
It should be noted that there may be reasons why the 'data' column is defined as 'text' type.
